setup.cfg

```diff
@@ -39,6 +39,7 @@ install_requires =
     pandas>1.0
     marshmallow>=3.7.1
     numpy>=1.18
+    whylabs-client==0.1
```

Comment:
nit: use `>=` here. If you are worry you can use `whylabs-client >=0.1, <= 0.2` so that if we make breaking changes to the API it's reflected in the client.